File: ch11/MyNotesPOOO/PersistenceLayer/domain/Note.m
//
//  Note.m
//  MyNotes
//
//  Created by MyZiyue on 2018/12/29.
//  Copyright © 2018 MyZiyue.com. All rights reserved.
//

#import "Note.h"

@implementation Note

-(instancetype)initWithDate:(NSDate *)date content:(NSString *)content {
    self = [super init];
    if(self) {
        self.date = date;
        self.content = content;
    }
    return self;
}

-(instancetype)init {
    self = [super init];
    if(self) {
        self.date = [[NSData alloc] init];
        self.content = @"";
    }
    
    return self;
}
@end

File: ch11/MyNotesPSSS/PersistenceLayer/domain/Note.swift
//
//  Note.swift
//  SMyNotes
//
//  Created by MyZiyue on 2018/12/29.
//  Copyright © 2018 MyZiyue.com. All rights reserved.
//

import Foundation

class Note {
    var date: NSDate!
    var content: NSString!
    
    init(date: NSDate, content: NSString) {
        self.date = date
        self.content = content
    }
    
    init() {
        self.date = NSDate()
        self.content = ""
    }
}
